cmd/snap-failure/cmd_snapd.go

```diff
@@ -145,6 +145,15 @@ func (c *cmdSnapd) Execute(args []string) error {
 	}
 
 	logger.Noticef("restoring invoking snapd from: %v", snapdPath)
+	if prevRev != "0" {
```

Comment:
maybe a comment explaining that this meas we have a previous revision?

Reply:
Done.